src/wheel/cli/unpack.py

```diff
@@ -18,6 +18,11 @@ def unpack(path: str, dest: str = ".") -> None:
         namever = wf.parsed_filename.group("namever")
         destination = Path(dest) / namever
         print(f"Unpacking to: {destination}...", end="", flush=True)
-        wf.extractall(destination)
+        for zinfo in wf.filelist:
+            wf.extract(zinfo, destination)
+
+            # Set permissions to the same values as they were set in the archive
```

Comment:
Linking to the CPython bug here might help for future helpful users that think they can clean this up. Or to educate people who didn't know this was a bug until now (like me) :)

Reply:
Done.